.nf-core.yml

```
@@ -1,8 +1,21 @@
-repository_type: pipeline
+bump_version: null
 lint:
   files_exist:
     - conf/igenomes.config
   nextflow_config:
     - config_defaults:
         - params.dfast_config
-nf_core_version: "2.14.1"
+nf_core_version: 3.0.2
+org_path: null
+repository_type: pipeline
+template:
+  author: Andreas Wilm, Alexander Peltzer
+  description: Simple bacterial assembly and annotation
+  force: false
+  is_nfcore: true
+  name: bacass
+  org: nf-core
+  outdir: .
+  skip_features: null
```

```suggestion
  skip_features: 
    - igenomes
```
Based on your lint settings.

---

after you add this value, these things will be handled automatically. so I would recommend to wait for another sync after this is merged (or do a manual sync after you added these changes `dev`, see https://nf-co.re/blog/2024/tools-3_0_0#tip-avoiding-merge-conflicts)

---

Got it, thanks ;)